ui-tests/cypress/support/restAPI/emailConfigSenderAPI.js

```diff
@@ -0,0 +1,61 @@
+const apiURL     = Cypress.config('restAPI');
+
+const controller = `${apiURL}plugins/emailSettings/senders`;
```

Comment:
Move the definition of the controller to the new `controllersEndPoints` file and import it back here, as it is, now, for the other controllers